src/main/java/at/ac/tuwien/kr/alpha/solver/heuristics/BranchingHeuristicFactory.java

```diff
@@ -41,19 +41,33 @@
 		NAIVE,
 		BERKMIN,
 		BERKMINLITERAL,
+		@Deprecated
```

Comment:
I think it would be good to have a comment somewhere, explaining why those heuristics are deprecated. Maybe a short sentence in the class-level javadoc?

Reply:
Good idea!